### PR TITLE
Adding logic to check what bridges there are.

### PR DESCRIPTION
**Pull request by Edenb2131**
Left to implement only the algorithm itself in NonDirectedGraph.cpp under find_bridges().

### CODE REVIEW

File: Summer Algorithms Assignment/Graph.cpp [hunk 1]
```diff
@@ -36,8 +36,14 @@ void graph::visit(vertex& i_vertex)
 	i_vertex.set_color(Color::GRAY);
 	for (auto& neighbor : neighbors)
 	{
-		vertex& real_neighbor = m_vertexes[neighbor.get_value() - 1]; //
-		mark_edge(i_vertex, real_neighbor);
+		vertex& real_neighbor = m_vertexes[neighbor.get_value() - 1];
+        if(real_neighbor.get_color() != Color::WHITE){
+            continue;
+        }
+        cout<< "Visiting vertex " + to_string(real_neighbor.get_value()) + " ";
```
Comment (Ronenii) [inline Summer Algorithms Assignment/Graph.cpp:45]: We were missing the #include of that allows the to_string. I added it

File: Summer Algorithms Assignment/NonDirectedGraph.cpp [hunk 2]
```diff
@@ -10,8 +10,8 @@ void non_directed_graph::set_edge(vertex& i_src, vertex& i_dst)
 
 void non_directed_graph::mark_edge(vertex& i_current_vertex, vertex& i_neighbor_vertex)
 {
-	i_current_vertex.get_neighbors().remove(i_neighbor_vertex);
-	i_neighbor_vertex.get_neighbors().remove(i_current_vertex);
+//	i_current_vertex.get_neighbors().remove(i_neighbor_vertex);
+//	i_neighbor_vertex.get_neighbors().remove(i_current_vertex);
```
Comment (Ronenii) [inline Summer Algorithms Assignment/NonDirectedGraph.cpp:14]: Why is this commented out?

**Review by Ronenii — COMMENTED** [review]
Overall looks good! I just want to know why is mark_edge() commented out and as far as I'm concerned we can merge this to master. Thanks!

**Review by Ronenii — APPROVED** [review]

File: Summer Algorithms Assignment/Graph.cpp (hunk 1 above)
Comment (Edenb2131) [reply Summer Algorithms Assignment/Graph.cpp:45]: Thank you.

**Comment by Edenb2131** [comment]
Mark edges deleted necessary edges. It was made for an eularian circle.

**Pull request merged**